.github/workflows/centos.yml

```diff
@@ -1,12 +1,18 @@
 name: Centos containers
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  # Allow to run this workflow manually from the Actions tab
+  workflow_dispatch:
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
-    strategy:
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name
+
+strategy:
```

Comment:
This indentation looks odd, at least changed from the previous version.

Reply:
Ah, an accidental change. Fixed in commit 343519a7f26f8df5c5dbdcd08f239c6bceb6f938.